src/Assignment-3/assignment3.css

```diff
@@ -0,0 +1,53 @@
+/* Our CSS file. */
+
+
+.container, .content {
+width: 100%;
+height: 100%;
+}
+.container1{
+    background-color: #F2F2F2;
+    border: 2px solid;
+    width: 100%;
+    height: 50%;
+    display: flex;
+    margin-bottom: 10px;
+}
+.container2{
+    background-color: #F2F2F2;
+    border: 2px solid;
+    width: 100%;
+    height: 40%;
+    margin-top: 40px;
+}
+.upperh1{
+    
+    width: 50%;
+    margin: 0px 20px 20px 20px  ;   
+}
+.lowerh1{
+    height: 50%;
+    text-align: center;
+    margin-bottom: 10px;
+    
+}
+#render-wiki{
+    border: 1px solid;
+    width: 50%;
+    margin: 20px;
+
+}
+#render-dictionry{
+    border: 2px solid;
+
+    margin-top: 10px;
+
+}
+button 
```

Comment:
increase the width and height of the button in both iframes divisions

Reply:
made the improvements on button styling